package.json

```diff
@@ -4,8 +4,7 @@
   "description": "A distributed networking stack for connecting peers",
   "main": "swarm.js",
   "dependencies": {
-    "@hyperswarm/discovery": "^1.1.0",
-    "@hyperswarm/network": "^1.1.2",
+    "@hyperswarm/network": "git+https://github.com/hyperswarm/network.git#adaptive-ephemerality",
```

Comment:
TODO update to release when https://github.com/hyperswarm/network/pull/18 is merged and released